### ISSUE TITLE
I need more than 100 revisions in history.

### ISSUE THREAD

**Issue by parkpd** [SRombauts/UE4GitPlugin#73]
Hi, I want to see all revisions in 'Source Control' -> 'History'.
100 revisions is sometimes not enough.
If it could hurts speed of getting history from SVC, making it with options could be OK.
Thanks.

**Comment by SRombauts** [comment]
Yes, as said previously, this could be increased, or made configurable, but this is not a priority for me right now
